### ISSUE TITLE
No template option under page attributes

### ISSUE THREAD

**Issue by JoannaMJohnson** [bhadaway/blankslate#60]
I love the BlankSlate template overall, a super pared down framework is exactly what I was looking for. However, I'm having an issue with the page templates - it is not showing as an option at all on the Page editor. Templates does display when I activate a different theme, but on this theme, these are the only options that show for me. Is there a way to get Page Template added back onto the editor?

![image](https://github.com/user-attachments/assets/792dc364-9979-4634-a769-0e1821ffe050)

**Comment by JoannaMJohnson** [comment]
I found the issue is resolved by adding a new custom template file into the theme.

**Comment by bhadaway** [comment]
Yes, BlankSlate really is blank, and contains no custom page templates. Glad you got it sorted.

**Comment by JoannaMJohnson** [comment]
Thanks! I do really love how pared down it is - most of the other 'skeleton' themes I tried out still came with a ton of extra code I didn't really want. :) I just didn't realize that Wordpress completely hid the page template selector if there were no custom templates added into the theme.

**Comment by bhadaway** [comment]
Yeah, there are a lot of features actually that you would assume are WordPress-side default features simply because they're ubiquitous in nearly all themes, and therefore were always there in your admin.

It's not specifically made for BlankSlate or anything, but this plugin might be useful to you if don't want to have to maintain custom templates yourself:

https://wordpress.org/plugins/blank/
